examples/text_labels_and_annotations/plot_tex.py

```diff
@@ -0,0 +1,39 @@
+"""
+=================================
+Rendering math equation using TeX
+=================================
+
+You can use TeX to render all of your matplotlib text if the rc
+parameter text.usetex is set.  This works currently on the agg and ps
+backends, and requires that you have tex and the other dependencies
+described at http://matplotlib.org/users/usetex.html
+properly installed on your system.  The first time you run a script
+you will see a lot of output from tex and associated tools.  The next
+time, the run may be silent, as a lot of the information is cached in
+~/.tex.cache
+
+Notice how the the label for the y axis is provided using unicode!
+
+"""
+from __future__ import unicode_literals
+import numpy as np
+import matplotlib
+matplotlib.rcParams['text.usetex'] = True
+matplotlib.rcParams['text.latex.unicode'] = True
+import matplotlib.pyplot as plt
+
+
+t = np.linspace(0.0, 1.0, 100)
+s = np.cos(4 * np.pi * t) + 2
+
+t = np.arange(0.0, 1.0 + 0.01, 0.01)
+s = np.cos(2*2*np.pi*t) + 2
+
+fig, ax = plt.subplots(figsize=(6, 4), tight_layout=True)
+ax.plot(t, s)
+
+ax.set_xlabel(r'\textbf{time (s)}')
```

Comment:
To avoid losing content due to the merge.  It is worth showing that the font size and colors work with latex along with our internal text rendreing.